src/shared/libmount-util.h

```diff
@@ -1,10 +1,43 @@
 /* SPDX-License-Identifier: LGPL-2.1+ */
 #pragma once
 
+#include <stdio.h>
+
 /* This needs to be after sys/mount.h */
 #include <libmount.h>
 
 #include "macro.h"
 
 DEFINE_TRIVIAL_CLEANUP_FUNC(struct libmnt_table*, mnt_free_table);
 DEFINE_TRIVIAL_CLEANUP_FUNC(struct libmnt_iter*, mnt_free_iter);
+
+static inline int libmount_parse(
```

Comment:
hmm, static inline? I think this should just be a regular function, it's complicated enough to warrant that.

Reply:
This was explained in the commit message that adds the function. I'll repush for the other things, and then I'll try to make the explanation clearer.